cirq/study/trial_result.py

```diff
@@ -108,7 +108,7 @@ def data(self) -> pd.DataFrame:
                 converted_dict[key] = [
                     value.big_endian_bits_to_int(m_vals) for m_vals in val
                 ]
-            self._data = pd.DataFrame(converted_dict)
+            self._data = pd.DataFrame(converted_dict, dtype=np.int64)
```

Comment:
Doesn't this limit us to 63 qubits?

Reply:
Pandas is actually smart about this. It'll use the suggested type when possible, but if you pass values that are too large, it'll upgrade to `dtype='object'`:

```
In [2]: pd.DataFrame({'a': []}, dtype=np.int64).to_numpy().dtype
Out[2]: dtype('int64')

In [3]: pd.DataFrame({'a': [1, 2]}, dtype=np.int64).to_numpy().dtype
Out[3]: dtype('int64')

In [4]: pd.DataFrame({'a': [1, 2**64+1]}, dtype=np.int64).to_numpy().dtype
Out[4]: dtype('O')
```

I've changed to `int` since it is less misleading.